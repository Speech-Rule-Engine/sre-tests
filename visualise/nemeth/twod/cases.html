<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<link type="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/npm/sre-visualiser/styles/style.css"/>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/d3/dist/d3.min.js"></script>
<link type="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/npm/sre-visualiser/styles/tree.css"/>
<script src="https://cdn.jsdelivr.net/npm/speech-rule-engine/lib/sre.js"></script>
<script async src="https://cdn.jsdelivr.net/npm/mathjax-full/es5/tex-mml-chtml.js"></script>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/sre-visualiser/lib/visualise.js"></script><title>Cases</title>
</head>

<body>
<h1>Cases</h1><div class="cell">
  <span class="counter">1</span>
  <span class="math">
     <math><mrow><mo>{</mo><mtable columnalign="left left" columnspacing="1em" rowspacing=".2em"><mtr><mtd><mi>x</mi><mo>=</mo><mi>y</mi></mtd></mtr><mtr><mtd><mn>5</mn><mi>x</mi><mo>&#x2212;</mo><mi>y</mi><mo>=</mo><mn>4</mn></mtd></mtr></mtable><mo></mo></mrow></math>
  </span>
  <span class="tree"></span>
</div>

</body>
<script type="text/javascript">
streeVis.HOVER_INFO.font = true;
streeVis.config.expanded = true;
streeVis.renderCells();
</script>
</html>