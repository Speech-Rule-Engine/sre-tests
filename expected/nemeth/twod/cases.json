{
  "factory": "braille2D",
  "name": "Nemeth2dCasesTests",
  "information": "Two Dimensional rules for case statements structures.",
  "locale": "nemeth",
  "domain": "default",
  "style": "default",
  "modality": "braille",
  "active": "Nemeth2D",
  "tests": {
    "case_16.4-1": {
      "input": "<mrow><mo>{</mo><mtable columnalign=\"left left\" columnspacing=\"1em\" rowspacing=\".2em\"><mtr><mtd><mi>x</mi><mo>=</mo><mi>y</mi></mtd></mtr><mtr><mtd><mn>5</mn><mi>x</mi><mo>&#x2212;</mo><mi>y</mi><mo>=</mo><mn>4</mn></mtd></mtr></mtable><mo></mo></mrow>",
      "expected": "⠨⠠⠷⠭⠀⠨⠅⠀⠽⠀⠀⠀⠀⠀\n⠨⠠⠷⠼⠢⠭⠤⠽⠀⠨⠅⠀⠼⠲"
    }
  }
}
